takwimu/templates/takwimu/_includes/navbar/_index.html

```diff
@@ -99,3 +100,17 @@
     </div>
   </div>
 </nav>
+
+<style>
+  .navbar.fixed-top .dropdown-menu-lg li[type="disc"] {
+	  color:#7ab1f2;
+  }
```

Comment:
This looks redundant to the style below. How are they different?

Reply:
This applies color to the type disc only, without the content in the anchor tag. I'll put this in the `ul` element instead as an inline style...